### ISSUE TITLE
cluster contraction

### ISSUE THREAD

**Issue by dmaier95** [voldemort/voldemort#376]
It's possible to reduce the number of nodes whitout a huge downtime?

**Comment by FelixGV** [comment]
Yes, both cluster expansion and shrinkage are online operations. There is an impact to the cluster because of data movement, and so you should monitor it carefully, but it is meant to be done without any down time nonetheless.

**Comment by dmaier95** [comment]
@FelixGV 

Thank you for your quick response. Do you have a short tutorial for cluster shrinkage?

**Comment by chiquiff** [comment]
Hey! I would really appreciate a tutorial for cluster shrink aswell! Thanks!
